isb_lib/geome_adapter.py

```diff
@@ -119,7 +119,7 @@ def recordsInProject(self, project_id, record_type):
                 expedition_modified_datetime = dateparser.parse(expedition_modified_date)
                 # For each expedition, check to see if the mod date is greater than the last time we fetched.
                 # If it is, fetch the bcids of the samples in the expedition.
-                if expedition_modified_datetime.timestamp() > self._date_start.timestamp():
+                if self._date_start is None or expedition_modified_datetime.timestamp() > self._date_start.timestamp():
```

Comment:
Another testing bug -- if we weren't doing a partial update we were crashing -- hit during manual testing on a full record fetch.